README.md

```diff
@@ -1,56 +1,39 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+OOP-School-Library
```

Comment:
- [ ] Please remove `Project Name` in line 3 and add `OOP-School-Library` instead to make your README file look more professional.
[Hint] if you do it like that:
`# OOP-School-Library`
it will look like that:
# OOP-School-Library